### ISSUE TITLE
How to hide labels and charts fill that position?

### ISSUE THREAD

**Issue by ghost** [indiespirit/react-native-chart-kit#683]
I am trying to hide my eyes and make the chart fill the width of the screen. Is there any way to do that?
![image](https://user-images.githubusercontent.com/82165349/236989041-25623ed1-c8bd-41c4-b8b3-25f1c4648c67.png)
This is my code
![image](https://user-images.githubusercontent.com/82165349/236989289-e785700a-280f-458e-93b9-4e241c5bea14.png)

**Comment by lovestaco** [comment]
` hideLegend={true}
`

**Comment by owattenmaker** [comment]
> hideLegend={true}

I don't believe this works for anything other than progress charts. i'd like to implement this on a bar or line graph

**Comment by lovestaco** [comment]
Oh sorry didn't know

**Comment by pritsawa** [comment]
https://www.youtube.com/watch?v=m8gyy7JItmU https://www.youtube.com/watch?v=1WZjeESH4a0
